var/spack/repos/builtin/packages/py-neo4j/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyNeo4j(PythonPackage):
+    """This is the neo4j bolt driver for python from the official repository"""
+
+    pypi = "neo4j/neo4j-5.25.0.tar.gz"
+
+    license("LGPL-3.0-only")
+
+    version("5.25.0", sha256="7c82001c45319092cc0b5df4c92894553b7ab97bd4f59655156fa9acab83aec9")
+
+    depends_on("py-pytz", type="run")
+    depends_on("py-setuptools@68.0.0", type="build")
+    depends_on("py-tomlkit@0.11.8", type="build")
```

Comment:
(FYI, no action) I can already tell this is going to be an annoying package :-) The devs seem to pin the build dependency to specific (ancient for setuptools) versions and then change them between releases... You'll end up with a lot of `depends_on` this `when` that version lines here.